src/main/java/core/basesyntax/services/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+
+    @Override
+    public void write(String report, File finalReportFile) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(finalReportFile))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file - " + finalReportFile, e);
+        }
+    }
+}
```

Comment:
The implementation of FileWriterService seems to be correct and follows the try-with-resources pattern to ensure the BufferedWriter is closed properly. However, please ensure that the FileWriterServiceImpl is placed in the correct package structure. According to the checklist, all service implementations should reside in the 'service.impl' package. If it's not the case, please move this class to the correct package.